test/suites/integration/expiration_test.go

```diff
@@ -54,7 +54,11 @@ var _ = Describe("Expiration", func() {
 		selector = labels.SelectorFromSet(dep.Spec.Selector.MatchLabels)
 	})
 	It("should expire the node after the expiration is reached", func() {
-		nodePool.Spec.Template.Spec.ExpireAfter = karpv1.MustParseNillableDuration("2m")
+		if env.IsDefaultNodeClassKWOK() {
+			nodePool.Spec.Template.Spec.ExpireAfter = karpv1.MustParseNillableDuration("30s")
+		} else {
+			nodePool.Spec.Template.Spec.ExpireAfter = karpv1.MustParseNillableDuration("5m")
```

Comment:
5 minutes seems a little long, right? I get why it can be shorter with KWOK, but any reason to increase it from 2 to 5 for non-KWOK NodeClasses?

Reply:
It's not a given that nodes will register and become healthy in 5 mins for all cloud providers. This would be to give more buffer

Reply:
I'd rather leave it at 2 minutes and increase as needed for other cloudproviders in the interest in keeping E2E tests fast.

Reply:
This from testing with the AWS Provider, and the 5 minites is needed to validate that the nodeclaim expired and new node is spun up.